docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -4,116 +4,159 @@ title: Cellular
-| Quectel BG95-M3          | ✔                  | ✔  | ✔       |
-| Quectel M95              | -                   | -   | ✔       |
+**Cat-M1**, **NB-IoT**, and **GSM** (Global System for Mobile Communications) are all cellular network technologies with distinct characteristics. **GSM**, the most widely used cellular standard globally, provides higher data rates compared to **NB-IoT** and **Cat-M1** but has higher power consumption. In contrast, **Cat-M1** and **NB-IoT** are optimized for IoT applications. **Cat-M1** offers higher data rates and mobility, while **NB-IoT** provides ultra-low power consumption and extended coverage. The choice among **GSM**, **Cat-M1**, and **NB-IoT** depends on the specific requirements of the application, considering factors such as data rates, power consumption, and coverage area.
 
+# Supported Cellular Modules
 
-**Cat-M1**, **NB-IoT**, and **GSM** (Global System for Mobile Communications) are all cellular network technologies with distinct characteristics. **GSM**, the most widely used cellular standard globally, provides higher data rates compared to **NB-IoT** and **Cat-M1** but has higher power consumption. In contrast, **Cat-M1** and **NB-IoT** are optimized for IoT applications. **Cat-M1** offers higher data rates and mobility, while **NB-IoT** provides ultra-low power consumption and extended coverage. The choice among **GSM**, **Cat-M1**, and **NB-IoT** depends on the specific requirements of the application, considering factors such as data rates, power consumption, and coverage area.
+The cellular network for the Meadow platform is compatible with various modules, each supporting different network operation modes. The following table describes the network operation modes supported by each module:
 
+| Modules / Network Modes   | Cat-M1 (LTE-M or eMTC) | NB-IoT | GSM/GRPS 2G | GNSS   |
+|---------------------------|------------------------|--------|-------------|--------|
+| Quectel BG95-M3           | ✔                     | ✔      | ✔           | ✔      |
+| Quectel M95               | -                      | -      | ✔           | -      |
 
-## Network configuration
+# Hardware configuration
 
-Using Cellular Meadow, you don't need to understand the complexities of AT commands or invest time studying module datasheets to enable the cellular network interface.
+## Quectel BG95M3 with NimbeLink Skywire click board
 
-You only need to create a **cell.config.yaml** file, where the cell settings will be defined, define the cell interface on the **meadow.config.yaml**, and reserve some pins used by the cell module.
+To setup the hardware, you could use a Skywire click adapter, which hosts NimbeLink/Skywire ™ cellular modems (using stacking headers) to MikroElektronika development boards. 
 
-Then, to configure your network you just need to follow these three steps:
+![Quectel BG95M3 with NimbeLink Skywire click board](images/modem-skywire-click.jpg)
```

Comment:
Not only that, but all the images seem missing in the PR [preview](https://github.com/WildernessLabs/Documentation/blob/patridge-update-cellular-from-develop/docs/Meadow/Meadow.OS/Cellular/index.md).

Reply:
I noticed that as well, though they are working as published on beta-developer. So, I figured it's mostly okay. Ideally, they would work in both systems, though.
https://beta-developer.wildernesslabs.co/Meadow/Meadow.OS/Cellular/#quectel-bg95m3-with-nimbelink-skywire-click-board
I'll leave it as is so that we can fix it properly on develop, hoping to avoid a conflict when that merge comes. (This may all be a conflict, though...not sure.)

Reply:
Oh...I'm catching on now. The image files need to come over with this PR as well. This is going to be much more complicated to pick updates then. I don't think the GitHub web UI is going to be sufficient.